### ISSUE TITLE
Notifications in Ubuntu

### ISSUE THREAD

**Issue by richard512** [uiucseclab/arpspoofdetect#2]
Something like this:

```
notify-send -u critical -i "software-update-urgent" 'ARP Spoofing Detected' '3c:15:c2:c0:45:ee (Spoofed)'
```

Pythonified:

```
import shlex, subprocess
args = shlex.split("notify-send -u critical -i 'software-update-urgent' 'ARP Spoofing Detected' '3c:15:c2:c0:45:ee (Spoofed)'")
subprocess.Popen(args)
```

**Comment by ninjatrench** [comment]
This looks promising, should be part of codebase. @uiucseclab

**Comment by richard512** [comment]
# Here's what it looks like:

![screenshot of notify-send](https://i.imgur.com/o6vty9n.png)

**Comment by ninjatrench** [comment]
@richard512 
I guess the repository is not actively maintained, not sure if PR will be merged anytime soon.

**Comment by richard512** [comment]
![screenshot of notify-send](https://memegenerator.net/img/instances/60483873/ill-just-wait-here-then.jpg)

**Comment by uiucseclab** [comment]
Its a student project. I can put it into an active repo if there is interest.

Sent from my iPhone

On Nov 23, 2017, at 02:23, Harsh Daftary <notifications@github.com<mailto:notifications@github.com>> wrote:


@richard512<https://github.com/richard512>
I guess the repository is not actively maintained, not sure if PR will be merged anytime soon.

**Comment by richard512** [comment]
It's a niche interest, but I think it's pretty cool. Could get popular if you post it somewhere like hackernews or a netsec community.

**Comment by uiucseclab** [comment]
Fair enough. Will consider. Technically its not my code but the students who wrote it. Will try to find them and get their opinions.

Sent from my iPhone

On Nov 23, 2017, at 17:39, richard512 <notifications@github.com<mailto:notifications@github.com>> wrote:


It's a niche interest, but I think it's pretty cool. Could get popular if you post it somewhere like hackernews or a netsec community.
